DOC-8925: Organize billing based on folders (#18194)
cockroachdb/docs: src/current/cockroachcloud/folders.md

1. In the dialog, select the destination to move the cluster to, then click **Next**.
1. Click **Move**.

> Alternatively, you can move a cluster to a new folder by browsing to the location of the destination folder, then clicking the the three-dots **Action** button and selecting **Move**.

Reviewer: ```suggestion
To move a cluster to a new folder from the **Clusters** page:

1. Bowse to the location of the destination folder.
1. Click the the three-dots **Action** button, then select **Move**.
```
To align with the section just above. WDYT?

{{site.data.alerts.callout_info}}

> Clusters are [billed]({% link cockroachcloud/billing-management.md %}#view-invoices) to the folder they reside in at the end of the billing period, even if they were moved between folders during that time.

Reviewer: ```suggestion
A cluster is [billed]({% link cockroachcloud/billing-management.md %}#view-invoices) to the folder it resides in at the end of the billing period, even if it was moved from one folder to another during the billing period.
```
